yo1k/tic_tac_toe/core/game.py

```diff
@@ -256,23 +259,23 @@ def __last_step(step: int) -> bool:
         return step == Board.size() ** 2 - 1
 
     @staticmethod
-    def __surrender(state: State):
+    def __surrender(state: State) -> None:
         assert state.phase is Phase.INROUND
```

Comment:
Add another assertion:
```python
# for more players this method would have been implemented quite differently
assert State.player_count() is 2
```

Reply:
Done.

Reply:
I left a comment in the example for a reason: I think we need to clarify this new assertion.

Reply:
Done.